app/app.go

```diff
@@ -187,8 +187,6 @@ var (
 
 type SifchainApp struct {
 	*baseapp.BaseApp
-	FEATURE_TOGGLE_MARGIN_CLI_ALPHA_SifchainApp
```

Comment:
@Jedi2002 same here if we get rid of that we are breaking with Margin additional struct required fields.

Reply:
Done